queries.sql

```diff
@@ -182,4 +182,12 @@ JOIN animals on visits.animal_id = animals.id
 JOIN species on animals.species_id = species.id
 WHERE vet.name = 'Maisy Smith'
 GROUP BY species.name, vet.name
-ORDER BY count DESC lIMIT 1;
\ No newline at end of file
+ORDER BY count DESC lIMIT 1;
+
+EXPLAIN ANALYZE SELECT COUNT(*) FROM visits where animal_id = 4;
+EXPLAIN ANALYZE SELECT * FROM visits where vet_id = 2;
+EXPLAIN ANALYZE SELECT * FROM owners where email = 'owner_18327@mail.com';
+
+CREATE INDEX animal_id_asc on visits(animal_id ASC);
+CREATE INDEX vet_id_asc on visits (vet_id ASC);
+CREATE INDEX email_asc on owners(email ASC);
```

Comment:
- [ ] Like the previous CR mentioned Any manipulation of tables should be done in the schema.sql file. Kindly move this piece of code to the schema.sql file.